src/elliptic/curves/ed25519.rs

```diff
@@ -584,17 +587,38 @@ mod tests {
 
     #[test]
     fn test_serdes_pk() {
-        let pk = GE::generator();
-        let s = serde_json::to_string(&pk).expect("Failed in serialization");
-        let des_pk: GE = serde_json::from_str(&s).expect("Failed in deserialization");
+        let mut pk = GE::generator();
+        let mut s = serde_json::to_string(&pk).expect("Failed in serialization");
+        let mut des_pk: GE = serde_json::from_str(&s).expect("Failed in deserialization");
         let eight = ECScalar::from(&BigInt::from(8));
         assert_eq!(des_pk, pk * &eight);
 
-        let pk = GE::base_point2();
-        let s = serde_json::to_string(&pk).expect("Failed in serialization");
-        let des_pk: GE = serde_json::from_str(&s).expect("Failed in deserialization");
-        let eight = ECScalar::from(&BigInt::from(8));
+        pk = GE::base_point2();
+        s = serde_json::to_string(&pk).expect("Failed in serialization");
+        des_pk = serde_json::from_str(&s).expect("Failed in deserialization");
         assert_eq!(des_pk, pk * &eight);
+
+        // deserialize serialization of bytes_str < 64 hex
+        s = "{\"bytes_str\":\"2c42d43e1a277e8f3d7d5aacde519c80b913341e425b624d867f790d1578e0\"}"
```

Comment:
where did this number came from ?

Reply:
Just an example of a point on the curve with less than 32 bytes.
This comes to check that the padding serialization indeed works with such points (and also that a deserialization of a padding serialization works).